src/basic/missing_socket.h

```diff
@@ -98,6 +99,10 @@ struct sockaddr_vm {
 #define IPV6_RECVFRAGSIZE 77
 #endif
 
+#ifndef IPPROTO_MPTCP
+#define IPPROTO_MPTCP 262
+#endif
+
```

Comment:
@yuwata thank you for the review! Sure, I didn't know which one to pick: they both include items from `netinet/in.h`, and because what I declared is used in the `socket()` call, I put it there :)